cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,288 @@
+        padding-left: unit( @grid_gutter-width / 2 / 18px, em );
+        padding-right: unit( @grid_gutter-width / 2 / 18px, em );
+
+        float: right;
+        position: relative;
+        z-index: 20;
+
+        background: @white;
+        border: none;
+        color: @black;
+        font-size: unit( 18px / @base-font-size-px, em );
+        pointer-events: auto;
+
+        &:focus {
+            outline: 1px dotted @black;
+        }
+
+        .cf-icon:before {
+            content: @cf-icon-search;
+            font-size: unit( 20px / 18px, em );
+        }
+
+        // Show "Search" text in trigger at tablet sizes.
+        .respond-to-min( @bp-sm-min {
+            &-label:before {
+                .webfont-medium();
+                content: 'Search';
+            }
+        } );
+
+        // Only show flyout and close trigger at tablet/mobile sizes.
+        .respond-to-max( @bp-sm-max, {
+            height: unit( 60px / 18px, em );
+            min-width: unit( 60px / 18px, em );
+            padding-top: @margin_half__em;
+            padding-bottom: @margin_half__em;
+
+            &[aria-expanded="true"] {
+                background: @gray-10;
+                border: 1px solid @gray-50;
+                border-bottom: none;
+                color: @black;
+
+                .cf-icon:before {
+                    content: @cf-icon-delete;
+                }
+
+                // Cover up the border edge of the flyout so that it is attached
+                // visually to the trigger button.
+                &:after {
+                    display: block;
+                    height: 1px;
+                    width: 100%;
+
+                    position: absolute;
+                    left: 0;
+                    bottom: -1px;
+
+                    content: '\a0';
+                    background: @gray-10;
+                }
+            }
+        } );
+
+        // Show "Close" text in trigger at tablet sizes.
+        .respond-to-range( @bp-sm-min, @bp-med-min, {
+            &[aria-expanded="true"] {
+                .m-global-search_trigger-label:before {
+                    content: 'Close';
+                }
+            }
+        } );
+
+        .respond-to-min( @bp-med-min {
+            &[aria-expanded="true"] {
+                display: none;
+            }
+        } );
+    }
+
+    &_content {
+        display: none;
+
+        padding-top: unit( @grid_gutter-width / 4 / @base-font-size-px, em );
+
+        &[aria-expanded="true"] {
+            display: block;
+
+            transform: translateY(0);
+            transition: transform 0.25s ease-out;
+            visibility: visible;
+        }
+
+        // Create flyout at table/mobile size.
+        .respond-to-max( @bp-sm-max, {
+            box-sizing: border-box;
+            width: 100%;
+
+            position: absolute;
+            left: 0;
+            // Offset for height of trigger button.
+            top: 60px;
```

Comment:
Since it's absolutely positioned it aligns with the button's container:

![screen shot 2016-01-22 at 12 24 57 pm](https://cloud.githubusercontent.com/assets/704760/12517905/45a0b932-c103-11e5-870c-1239497cbbde.png)